azure-pipelines.yml

```
@@ -6,8 +6,8 @@ variables:
   NugetPackageVersionCamera: '$(CurrentSemanticVersion)'
   NugetPackageVersionMediaElement: '$(CurrentSemanticVersion)'
   NugetPackageVersionMaps: '$(CurrentSemanticVersion)'
-  TOOLKIT_NET_VERSION: '8.0.303'
-  LATEST_NET_VERSION: '8.0.x'
+  TOOLKIT_NET_VERSION: '9.0.100-rc.2.24474.11'
```

Reminder to remove prerelease version before merging

```suggestion
  TOOLKIT_NET_VERSION: '9.0.100'
```